src/js/routes/VoterGuide/UnknownTwitterAccount.jsx

```diff
@@ -17,7 +17,7 @@ export default class UnknownTwitterAccount extends Component {
 
   render () {
     renderLog(__filename);
-    const { twitter_handle: twitterHandle, twitter_name: twitterName } = this.props;
+    const { twitterHandle: twitterHandle, twitterName: twitterName } = this.props;
```

Comment:
Thank you @fulin426, when deconstructing props or the state, if the variable is already in the format you want it in, no reaassignment is needed. That is, you don't need to write `twitterHandle: twitterHandle`. I will update this now.